src/MAVN.Service.PartnerManagement.DomainServices/Helpers/DoubleExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Globalization;
+
+namespace MAVN.Service.PartnerManagement.DomainServices.Helpers
+{
+    public static class DoubleExtensions
+    {
+        public static string ToDotString(this double value)
```

Comment:
Since this is used only in GeocodingRestService, let's move this there as a private method